test/unittests/utils/test_pet.py

```diff
@@ -116,3 +116,23 @@ def test_read_psf_information(tmp_path: os.PathLike, psf_df: pd.DataFrame):
         ["ses-M00", "ses-M18"],
         "FDG",
     ) == [[8, 9, 10], [8, 9, 10]]
+
+
+@pytest.mark.parametrize(
+    "label", ["pons", "cerebellumPons", "pons2", "cerebellumPons2"]
+)
+def test_get_suvr_mask(label: str):
+    from clinica.utils.pet import get_suvr_mask
+
+    assert get_suvr_mask(label).exists()
```

Comment:
"Unresolved attribute reference 'exists' for class 'PathLike'"

```suggestion
    assert Path(get_suvr_mask(label)).exists()
```